### PR TITLE
Ad-remove lists

### PR DESCRIPTION
**Pull request by efoosag**
Included add and remove functionality to my to-do list project

### CODE REVIEW

File: src/print.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function printMe() {
+  console.log('I get called from print.js!');
+}
```
Comment (danmainah) [inline src/print.js]: - [ ] KIndly remove this file, it doesn't serve any purpose in this project also its bad practise to have console.log in your codebase.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+let taskDescription = '';
+const taskCompleted = '';
+
+const loadListElement = () => {
+  const li = document.createElement('li');
+  li.classList.add(
+    // 'list-group-item',
+    // 'justify-content-between',
+    // 'd-flex',
+    // 'align-items-center',
```
Comment (danmainah) [inline src/index.js]: - [ ] Kindly ensure to remove the commented code, it's not good practice to have commented code in your files.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        } else {
+          item.nextElementSibling.classList.remove('completed');
+          elem.completed = false;
+        }
+      }
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    });
+  });
+});
+
```
Comment (danmainah) [inline src/index.js:59]: - [ ] Kindly check your task completing function, whenever I mark a task completed the changes made are not stored in the local storage.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @efoosag, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by dasileker — APPROVED** [review]
### **Hi @efoosag,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**